src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ModifiersProvider.java

```diff
@@ -137,9 +137,9 @@ default boolean isAbstract() {
     }
 
     /**
-     * Checks that the method is concrete and not abstract.
+     * Checks that this element is concrete and not abstract.
```

Comment:
It might be worth clarifying that we don't mean `isAbstract()` here.  We specifically mean that it corresponds to a method with a real implementation or a type which can be instantiated.